nemo_skills/pipeline/generate.py

```diff
@@ -207,8 +210,11 @@ def generate(
     if generation_module is None:
         generation_module = GENERATION_MODULE_MAP[generation_type or GenerationType.generate]
 
-    generation_task = importlib.import_module(generation_module)
-    if not hasattr(generation_task, 'GENERATION_TASK_CLASS'):
+    if os.sep in generation_module:
```

Comment:
Can you please do the same in here https://github.com/NVIDIA/NeMo-Skills/blob/main/nemo_skills/pipeline/utils/eval.py#L365?